java-bookstore/src/main/java/com/lambdaschool/bookstore/config/ResourceServerConfig.java

```diff
@@ -34,11 +34,12 @@ public void configure(HttpSecurity http) throws Exception
                                    "/configuration/ui",       // swagger
                                    "/configuration/security", // swagger
                                    "/swagger-ui.html",        // swagger
-                                   "/webjars/**"              // swagger
+                                   "/webjars/**",              // swagger
+                                    "/**"
                         ).permitAll()
                 // hasAnyRole can be a list of roles as in "ADMIN", "DATA"
-                .antMatchers("/roles", "/actuator/**").hasAnyRole("ADMIN")
-                .antMatchers("/**").authenticated()
+                .antMatchers("/roles", "/actuator/**").hasAnyRole("ADMINS")
```

Comment:
I see you have antmatchers set up to restrict access to certain endpoints for admins, but remember that you have two other roles as well!